### ISSUE TITLE
Problems for Windows users

### ISSUE THREAD

**Issue by TomDeneire** [melvinwevers/CV_tutorial#1]
While installing this a number of Windows users had problems. Apart from certain commands not working in the command line or Powershell ('source' for instance) which is of course to be expected, a major issue was that the path `virtualenvs/cv_course/bin` did not exist. There was a directory `virtualenvs/cv_course/bin/scripts` though, which contained the file `activate.bat`. This got us through the process. (Also there were pip issues; `pip` being needed instead of `pip3`, but that's probably on us)

**Comment by melvinwevers** [comment]
I'm going to look into this.

**Comment by gnmarten** [comment]
I am not a specialist at all, but in my mixed Windows/Linux experience, the "conda create -n CV_tutorial --file req.txt" command did not work, I asume because the required packages in the req.txt file are hard-coded to osx-64 (MAC) versions, which gives the error message that these packages are not available: "PackagesNotFoundError: The following packages are not available from current channels". My noob solution was to manually select all packages in Anaconda Navigator, because they are actually available indeed.

**Comment by melvinwevers** [comment]
@gnmarten You are correct that they are hard-coded to osx-64. I'll look into making it more generic. 

The noob solution was going to be my fallback for the workshop ;)
